apps/parsley/src/components/ErrorHandling/Sentry.tsx

```diff
@@ -30,6 +42,17 @@ const initializeSentry = () => {
 
 const isInitialized = () => !!getClient();
 
+const processHtmlAttributes = (htmlElement: HTMLElement) => {
```

Comment:
nit. can we add a comment and is there any way we can test this and the beforeBreadcrumb hook?